src/main/java/seedu/fast/logic/commands/AppointmentCommand.java

```diff
@@ -65,7 +70,8 @@ public AppointmentCommand(Index index, Appointment appointment) {
     public CommandResult execute(Model model) throws CommandException {
         List<Person> lastShownList = model.getFilteredPersonList();
 
-        if (index.getZeroBased() >= lastShownList.size()) {
+        if (CommandUtil.checkIndexExceedLimit(index, lastShownList)) {
+            logger.warning("-----Invalid Add Appointment Command: Invalid Index-----");
```

Comment:
Should this string be declared as a constant to make clear its purpose to readers? Similar cases below.

Reply:
Similar cases in other classes also